src/NzbDrone.Core/Notifications/Xbmc/XbmcSettings.cs

```diff
@@ -65,7 +66,7 @@ public XbmcSettings()
         public bool AlwaysUpdate { get; set; }
 
         [JsonIgnore]
-        public string Address => $"{Host.ToUrlHost()}:{Port}";
```

Comment:
`Address` is only used for logging, https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/Notifications/Xbmc/XbmcJsonApiProxy.cs#L78 is where we use it for the actual connection. We either need to make the default `/jsonrpc` or combine them there. I think making the default is the better route, in the future if they add a different API then we could just add it as a separate connection.